common/app/conf/switches.scala

```diff
@@ -287,6 +287,10 @@ object Switches {
     "If this switch is on, commercial components will be fed by job feed.",
     safeState = Off, sellByDate = never)
 
+  val MembersAreaSwitch = Switch("Commercial", "gu-members-area",
+    "If this switch is on, content flagged with membershipAccess will be protected",
+    safeState = On, sellByDate = never)
```

Comment:
We don't generally start things off with a `never` sell by date. Rather we choose a sensible date at which to end the test and review.